packages/gateway/src/gateway.js

```diff
@@ -236,6 +236,7 @@ function getHeaders(request) {
     'X-Forwarded-For': `${request.headers.get(
       'cf-connecting-ip'
     )}${existingProxies}`,
+    'X-Requested-From': 'nft.storage',
```

Comment:
Perhaps we should use [`X-Forwarded-Host`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host)? WDYT?